libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationActivity.java

```diff
@@ -118,6 +118,7 @@ private void extractConfiguration(NavigationViewOptions.Builder options) {
     MapboxNavigationOptions navigationOptions = MapboxNavigationOptions.builder()
       .unitType(preferences.getInt(NavigationConstants.NAVIGATION_VIEW_UNIT_TYPE,
         NavigationUnitType.TYPE_IMPERIAL))
```

Comment:
It'd be cool to get a default unit type based on the local here (if one isn't set in the preferences) with something like https://stackoverflow.com/questions/4898237/using-locale-settings-to-detect-wheter-to-use-imperial-units. 

Current catch is the unit type isn't currently nullable in `MapboxNavigationOptions`, so we can switch that to `@Nullable` and if someone doesn't set it, we dont set the preference and get the default here when setting the default value for the `SharedPreference`